src/app/Shared/Services/Api.service.tsx

```diff
@@ -1117,6 +1118,34 @@ export class ApiService {
     );
   }
 
+  downloadDashboardLayout(layout: DashboardLayout): void {
+    const serializedLayout = this.getSerializedDashboardLayout(layout);
+    const filename = `cryostat-dashboard-${layout.name}.json`;
+    const file = new File([serializedLayout], filename);
+    const resourceUrl = URL.createObjectURL(file);
```

Comment:
There is a `createBlobURL ` in `@app/utils/utils` to handle this. We can reuse it here.